pkg/armrpc/asyncoperation/worker/service.go

```diff
@@ -21,9 +21,9 @@ import (
 
 	manager "github.com/radius-project/radius/pkg/armrpc/asyncoperation/statusmanager"
 	"github.com/radius-project/radius/pkg/armrpc/hostoptions"
-	"github.com/radius-project/radius/pkg/ucp/dataprovider"
+	"github.com/radius-project/radius/pkg/ucp/databaseprovider"
 	queue "github.com/radius-project/radius/pkg/ucp/queue/client"
-	qprovider "github.com/radius-project/radius/pkg/ucp/queue/provider"
+	"github.com/radius-project/radius/pkg/ucp/queue/queueprovider"
```

Comment:
Like here the **queueprovider** is under **ucp/queue**. I believe that this can be renamed to **ucp/queue/provider** but it is not in the scope of this PR.

Reply:
I'm planning to rearrange these in the next PR 👍